packages/app/src/composables/useTokenLibrary.ts

```diff
@@ -48,6 +50,11 @@ export default (context = useContext()) => {
     isRequestFailed.value = false;
     try {
       tokens.value = await retrieveTokens(context);
+      const arr = tokens.value.filter((e) => e.l1Address === ETH_TOKEN_L1_ADDRESS);
```

Comment:
please choose a more meaningful variable name.

```suggestion
      const ethToken = tokens.value.filter((e) => e.l1Address === ETH_TOKEN_L1_ADDRESS);
```